src/Engine/Objects/Actor.cpp

```diff
@@ -1129,7 +1129,7 @@ void Actor::AI_MeleeAttack(unsigned int uActorID, Pid sTargetPid,
         pActors[uActorID].speed.x = 0;
         pActors[uActorID].UpdateAnimation();
     } else {
-        Actor::AI_Pursue1(uActorID, sTargetPid, grng->random(2), 64, arg0);
+        Actor::AI_Pursue1(uActorID, sTargetPid, grng->random(2), 64_ticks, arg0);
```

Comment:
Ticks? Not 500 msecs?

Reply:
Just adding _ticks everywhere for now. Will propagate the types first, then think about changing places like this one in bulk.